sicario.c

```diff
@@ -155,28 +231,51 @@ int keyboardHit() {
 void drawCharWithOffset(int x, int y, char *c) {
 	mvprintw(y + OFFSET_HEIGHT,x + OFFSET_WIDTH, c);
 }
+void settingGameBoard() {	
 
-void settingGameBoard() {
 	clear();
+	decreaseGameBoardCount = 0;
 	for (int i = 0; i < BOARD_WIDTH; ++i){
 		for (int j = 0; j < BOARD_HEIGHT; ++j){
 			gameBoard[i][j] = 0;
 		}
 	}
-	for (int i = 0; i < BOARD_WIDTH; ++i){
-		gameBoard[i][0] = gameBoard[i][BOARD_HEIGHT - 1] = '#';
-		drawCharWithOffset(i, 0, "#");
-		drawCharWithOffset(i, BOARD_HEIGHT - 1, "#");
-		delay(10);
+	decreaseGameBoardSize();
```

Comment:
This function decrease the game board size, or show the game board or both? Be careful, functions should only do **one thing** and it name should **represent well** the thing that it does.